CMakeLists.txt

```
@@ -512,7 +513,28 @@ endif(COAL_HAS_OCTOMAP)
 add_subdirectory(doc)
 add_subdirectory(src)
 if(BUILD_PYTHON_INTERFACE)
-  add_subdirectory(python)
+  add_custom_target(${PROJECT_NAME}_python)
+  set_target_properties(
+    ${PROJECT_NAME}_python
+    PROPERTIES EXCLUDE_FROM_DEFAULT_BUILD True
+  )
+  if(COAL_PYTHON_NANOBIND)
+    if(CMAKE_VERSION VERSION_LESS 3.18)
+      set(DEV_MODULE Development)
+    else()
+      set(DEV_MODULE Development.Module)
+    endif()
+    find_package(Python REQUIRED COMPONENTS Interpreter ${DEV_MODULE})
```

```suggestion
    find_package(Python REQUIRED COMPONENTS Interpreter Development.Module)
```



we can target CMake 3.22, ref. https://github.com/jrl-umi3218/jrl-cmakemodules#supported-cmake-versions

---

Oh yes, we've removed support for the old ones since this PR was initially created! Will fix this asap. Thanks